### PR TITLE
Production settings updated.

### PR DESCRIPTION
**Pull request by isht3**

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@isht3 I think #103 is the same with this PR? If so, please close anyone of them.

**Comment by isht3** [comment]
@RishabhJain2018 It was, thank you 👍

File: evalai/utils/config.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 AUTH_TOKEN_PATH = os.path.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
 
-API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://evalapi.cloudcv.org')
```
Comment (RishabhJain2018) [inline evalai/utils/config.py]: Please change it to `https://evalapi.cloudcv.org`

Comment (isht3) [reply evalai/utils/config.py]: @RishabhJain2018 Done!

**Comment by isht3** [comment]
@RishabhJain2018 Can we merge this?

**Pull request merged**